apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md

```diff
@@ -0,0 +1,68 @@
+---
+heading: Dialtone 9 released and published with @next tag
+author: Tico Ortega
+posted: '2023-12-28'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hi everyone, hope you're having a great holidays. As the year comes to an end, it's time for a big update!.
+In this blog post we'll talk about the release of Dialtone 9.
+
+## TLDR; Dialtone 9 released and published with @next tag
+
+We decided to release a mono-package containing all dialtone packages under the name of `@dialpad/dialtone`.
+
+This aligns better with our monorepo approach, helps up to ease the task of maintaining all our packages up-to-date
+on clients using dialtone and makes using multiple dialtone packages easier.
+
+In order to do so, we increased the version of `@dialpad/dialtone` as this is a breaking change but released to
+@next tag to avoid unintentionally updating clients to this version of dialtone yet.
+
+Also, we're going to continue releasing Dialtone CSS library to `@dialpad/dialtone-css` (formerly dialtone) as a
+separate
+package in case someone still needs to use it.
+
+As soon as we can start, we're going to do the migration on product to update the dialtone version and migrate
+the usage of multiple dialtone packages to the mono-package with its multiple entries.
+
+### How do I update to Dialtone 9
+
+In order to migrate to dialtone 9, first remove the separate dialtone packages:
+
+```shell
+npm uninstall @dialpad/dialtone-vue @dialpad/dialtone-icons @dialpad/dialtone @dialpad/eslint-plugin-dialtone
+```
+
+*This will uninstall all the existent separate dialtone packages that are included on the mono-package.*
+
+Next, install the `@dialpad/dialtone@next` package
+
+```shell
+npm install @dialpad/dialtone@next
+```
+
+Once it is installed, replace the paths of the old packages imports with the updated paths, e.g:
+
+Replace
+
+```less
+@import (inline) '@dialpad/dialtone-tokens/dist/css/variables-light.css';
+```
+
+With
+
+```less
+@import (inline) '@dialpad/dialtone/dist/tokens/css/variables-light.css';
+```
+
+Please refer to [Import packages](../../../../../../README.md#import-packages) for reference on how to import
```

Comment:
This does not direct me to the 'Import Packages' section.

Reply:
Once merged, it will work correctly.

You can try it changing the branch on GitHub, I think it will not work directly on the diff.

https://github.com/dialpad/dialtone/blob/docs/update-monopackage-documentation/apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md